ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -528,13 +548,22 @@ def _calc_qualified_demand(self):
                     rec.qualified_demand += demand_by_days[date]
         return True
 
+    @api.multi
+    def _calc_incoming_qty(self):
+        for rec in self:
+            rec.incoming_qty = 0.0
+            domain = rec._search_stock_moves_incoming_domain()
+            moves = self.env['stock.move'].search(domain)
+            rec.incoming_qty = moves.mapped('product_qty')
```

Comment:
Seems mapped return some error sometimes, alternative using for move in moves is working for me properly, need more testing.